src/ObjCRuntime/Runtime.cs

```diff
@@ -1806,6 +1806,21 @@ internal static IntPtr AllocGCHandle (object value)
 		{
 			return GCHandle.ToIntPtr (GCHandle.Alloc (value));
 		}
+
+#if __MACCATALYST__
+		static string _iOSSupportVersion;
+		internal static string iOSSupportVersion {
+			get {
+				if (_iOSSupportVersion == null) {
+					// This is how Apple does it: https://github.com/llvm/llvm-project/blob/62ec4ac90738a5f2d209ed28c822223e58aaaeb7/lldb/source/Host/macosx/objcxx/HostInfoMacOSX.mm#L100-L105
+					using (var dict = NSMutableDictionary.FromFile ("/System/Library/CoreServices/SystemVersion.plist")) {
+						_iOSSupportVersion = dict.ObjectForKey ((NSString) "iOSSupportVersion").ToString ();
```

Comment:
that `NSString` could be collected before being used
https://github.com/xamarin/xamarin-macios/issues/10146

Reply:
Fixed.